modules/Kconfig.tfm

```diff
@@ -3,6 +3,20 @@
 # Copyright (c) 2019, 2020 Linaro Limited
 # SPDX-License-Identifier: Apache-2.0
 
+
+config TFM_BOARD
+	string
+	default "nordic_nrf/nrf9160dk_nrf9160" if BOARD_NRF9160DK_NRF9160NS
+	default "nordic_nrf/nrf5340dk_nrf5340_cpuapp" if BOARD_NRF5340DK_NRF5340_CPUAPPNS
+	default "nordic_nrf/nrf5340pdk_nrf5340_cpuapp" if BOARD_NRF5340PDK_NRF5340_CPUAPPNS
+	default "nxp/lpcxpresso55s69" if BOARD_LPCXPRESSO55S69_CPU0
+	default "mps2/an521" if BOARD_MPS2_AN521
```

Comment:
The corresponding `board.cmake` line should be removed:
https://github.com/zephyrproject-rtos/zephyr/blob/d3ef29af44166eccb8620430c7b751fe5209cee1/boards/arm/mps2_an521/board.cmake#L4

Reply:
Thanks, removed.